ui/ts/pages/settings.ts

```diff
@@ -0,0 +1,116 @@
+// source: pages/account.ts
+/// <reference path="../../bower_components/mithriljs/mithril.d.ts" />
+/// <reference path="../components/topbar.ts"/>
+/// <reference path="../components/navbar.ts"/>
+/// <reference path="../util/convert.ts"/>
+/// <reference path="../models/sqlquery.ts" />
+/// <reference path="../models/helpus.ts" />
+
+// Author: Max Lang (max@cockroachlabs.com)
+
+/**
+ * AdminViews is the primary module for Cockroaches administrative web
+ * interface.
+ */
+module AdminViews {
+  "use strict";
+  import NavigationBar = Components.NavigationBar;
+  import Target = NavigationBar.Target;
+
+  /**
+   * Log is the view for exploring the logs from nodes.
+   */
+  export module Settings {
+    /**
+     * Page displays log entries from the current node.
+     */
+
+     let isActive: (targ: NavigationBar.Target) => boolean = (t: NavigationBar.Target) => {
+       return ((m.route.param("detail") || "") === t.route);
+     };
+
+    export module Page {
+      import MithrilVirtualElement = _mithril.MithrilVirtualElement;
+      import UserOptIn = Models.HelpUs.UserOptIn;
+
+      class HelpUsController {
+        userData: UserOptIn = new UserOptIn();
```

Comment:
It should be fine to make this a global. Right now, the modal and the page get a different copy of the UserOptIn().